pkg/sriov/sriov.go

```diff
@@ -70,6 +70,32 @@ func (p *pciUtilsImpl) GetPciAddress(ifName string, vf int) (string, error) {
 	return utils.GetPciAddress(ifName, vf)
 }
 
+// RebindVf unbind then bind the vf
+func (p *pciUtilsImpl) RebindVf(pfName, vfPciAddress string) error {
+	pfHandle, err := sriovnet.GetPfNetdevHandle(pfName)
+	if err != nil {
+		return err
+	}
+	var vf *sriovnet.VfObj
+	found := false
+	for _, vfObj := range pfHandle.List {
+		if vfObj.PciAddress == vfPciAddress {
+			vf = vfObj
+			found = true
+		}
+	}
+	if !found {
+		return fmt.Errorf("failed to find VF %s for PF %s", vfPciAddress, pfName)
+	}
+	if err = sriovnet.UnbindVf(pfHandle, vf); err != nil {
```

Comment:
in general is kinda wierd that you need the PF name to unbind/bind the VF from the driver.

for example for mellanox nics using mlx5 driver you just:
echo [VF PCI D:B:D.F address] > /sys/bus/pci/drivers/mlx5_core/unbind
echo [VF PCI D:B:D.F address] > /sys/bus/pci/drivers/mlx5_core/bind

Reply:
Opened issue for it https://github.com/Mellanox/sriovnet/issues/26